### PR TITLE
Review and update of lab1

### LINKED ISSUE #938: Updates required for lab1
It looks like our KubeVirt labs are out of date :(
It would be great if someone could run through the labs and update the guide accordingly. This is an opportunity to also increase the amount of information if need be.  

Github link: https://github.com/kubevirt/kubevirt.github.io/blob/main/labs/kubernetes/lab1.md
Website link: https://kubevirt.io/labs/kubernetes/lab1.html

Now that we're on a slower release cadence, we should be able to update these labs on an annual basis, tied to the most recently released version, which will be supported for ~12 months. Running and updating the labs could be a useful exercise for new contributors, provided they are supported if and when they need to troubleshoot changes in the procedure.

### PR DESCRIPTION
**Pull request by seb54000**
**What this PR does / why we need it**:
Review the lab1 tutorial, verify it is still working and add some details about vm / vmi difference

**Does this PR fix any issue?** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:

> Fixes #938

**Special notes for your reviewer**: 
THis is part fo the Kubecon 2024 Paris Contribfest

### CODE REVIEW

File: labs/kubernetes/lab1.md [hunk 1]
```diff
@@ -90,6 +96,8 @@ Connect to the serial console of the Cirros VM. Hit return / enter a few times a
 
 Disconnect from the virtual machine console by typing: `ctrl+]`.
 
+If you like to see the complete boot sequence logs form the console. You need to connect to serail console just after starting the vm (you can test by stopping and starting the vm again, see below).
```
Comment (aburdenthehand) [inline labs/kubernetes/lab1.md:99]:
```suggestion
If you like to see the complete boot sequence logs from the console. You need to connect to the serial console just after starting the VM (you can test this by stopping and starting the VM again, see below).
```

File: labs/kubernetes/lab1.md [hunk 2]
```diff
@@ -74,12 +76,16 @@ kubectl patch virtualmachine testvm --type merge -p \
...
-Now that the Virtual Machine has been started, check the status. Note the `running` status.
+Now that the Virtual Machine has been started, check the status (`kubectl get vms`). Note the `Running` status.
+
+You now want to see the instance of the vm you just started :
 
 ```
 {% include scriptlets/lab1/05_verify_testvm_instance.sh -%}
 ```
 
+Note the difference between vm (virtual machine) resource and vmi (virtual machine instance) resource. vmi didn't exists before starting the vm and vmi will be deleted when you'll stop the vm. (Also note that restart of the vm is needed if you like to change some properties. Just modifying vm is not sufficient, vmi has to be replaced.)
```
Comment (aburdenthehand) [inline labs/kubernetes/lab1.md:87]:
```suggestion
Note the difference between VM (virtual machine) resource and VMI (virtual machine instance) resource. The VMI does not exist before starting the VM and the VMI will be deleted when you stop the VM. (Also note that restart of the VM is needed if you like to change some properties. Just modifying VM is not sufficient, the VMI has to be replaced.) 
```

**Review by aburdenthehand — COMMENTED** [review]
Thanks for your help @seb54000 
It was very nice to work on this with you at our Contribfest.
I have made a couple of minor suggestions with regard to spelling, caps for VM and VMI and some light sentence structure.
After you accept these, you can squash all of the commits like we did last week, except squash them into your first commit (the unsigned one). After which you can run: `git commit --amend --signoff --no-edit` and force push to your branch. (This was the command I couldn't think of on Friday but will solve the lack of sign-off in your one commit after you've squashed them.)

File: labs/kubernetes/lab1.md (hunk 2 above)
Comment (seb54000) [reply labs/kubernetes/lab1.md:87]: Thanks a lot @aburdenthehand, it was a real pleasure meeting you and all other Kubevirt contributors. Your proposal of modifications are perfect, I will include them and solve this signing issue hopefully through squashing.

**Comment by xpivarc** [comment]
@aburdenthehand PTAL

File: labs/kubernetes/lab1.md [hunk 3]
```diff
@@ -90,6 +96,8 @@ Connect to the serial console of the Cirros VM. Hit return / enter a few times a
 
 Disconnect from the virtual machine console by typing: `ctrl+]`.
 
+If you like to see the complete boot sequence logs from the console. You need to connect to the serial console just after starting the VM (you can test this by stopping and starting the VM again, see below).
```
Comment (tiraboschi) [inline labs/kubernetes/lab1.md:99]: now we also have the capability to get guest console logs streamed out to the k8s logging system,
see: https://github.com/kubevirt/kubevirt/blob/055c6e0491fa93befa6372ca4d367916cabcb5af/staging/src/kubevirt.io/api/core/v1/schema.go#L466-L470

**Comment by aburdenthehand** [comment]
/remove-lifecycle stale
@tiraboschi Does it make sense to merge this so that the lab is updated and this work is not lost and then subsequently include the logging improvement?

**Comment by aburdenthehand** [comment]
/approve
I haven't heard back from @tiraboschi but I would rather one improvement be merged followed by a possible second rather than risk having no improvements :) 
@brianmcarey PTAL?

**Review by aburdenthehand — APPROVED** [review]

**Pull request merged**